deltachat-jsonrpc/src/api.rs

```diff
@@ -708,6 +708,19 @@ impl CommandApi {
         ChatId::new(chat_id).get_encryption_info(&ctx).await
     }
 
+    /// Get encryption info for a chat.
+    async fn get_chat_encryption_info_json(
```

Comment:
In fact this doesn't return JSON, it's serilized to JSON later. I suggest to rename this to `get_chat_encryption_info()` because it returns `EncryptionInfo`, and the old function to `get_chat_encryption_info_str()` or `get_chat_encryption_info_txt()`.